uiowa/uiowa
# Remove duplicate profiles `<title>` element.

# How to test

- Go to https://education.dev.drupal.uiowa.edu/directory and inspect meta tags like title, description, canonical and schema as you navigate between profiles and back to the directory listing.
- Compare the source code, not inspector, of the following URLs:
  - https://education.uiowa.edu/directory/john-achrazoglou
    - Notice the duplicate, empty `<title>` element.
  - https://education.dev.drupal.uiowa.edu/directory/john-achrazoglou
    - Confirm that the dev URL shows the correct meta tags: one title, description, canonical and schema while viewing source.
- Also, you can run some dev profile URLs through https://tamethebots.com/tools/fetch-render?utm_source=pocket_mylist and confirm those metatags are seen by bots. You need to uncheck "Honour robots.txt" and "Flatten DOM".
